### PR TITLE
Fix unity event manager and use update_locations endpoint to set heading

### PR DESCRIPTION
**Pull request by cburnout**
# Description
This fixes the event manager so it can figure out its user. right now its a string in the telemetry client but should eventually be moved out to its own thing(I am not sure what the looks like so its like this for now).
I had to do some less than ideal stuff to make the db client work in a way that was stable. This should also be fixed once we understand the issues there.  

Fixes # (issue)

## Bug Fix or Feature

- [ ] Bug
- [x] Feature
- [ ] Other

# Testing Instructions

- [x] Launch all of the server containers
- [x] start the scene and see that it doesn't update
- [x] Check that the user is added to the db using db tool (DBeaver is what I use)
- [x] stop the scene
- [x] start the scene again
- [x] start the python script to test
- [x] start the BearingHalo scene and see that the heading slowsly spins
- [x] turn off the the script
- [x] use the api to update the heading for the user_id and see that it updates in the scene
- [x] turn scene off and on and see that after it finds the right user_id it works again

## Linting

- [x] I've linted my code

### CODE REVIEW

**Review by isaacdenny — COMMENTED** [review]
Everything works great! just some rogue debugs and imports, and some extra syntax bits. We should start using black soon...

**Review by isaacdenny — CHANGES_REQUESTED** [review]
Heres the things to fix 

[ ] excess debugs (see comments)
[ ] extra lines in database.py
[ ] unused imports in database.py

File: ARSIS-Telemetry-GroundControl/telemetry-api/app/database.py [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+
 import psycopg2
+from psycopg2 import errors
 
 time.sleep(10)
 connection = None
 
+
 try:
+
```
[the remark this answers is not in the document]
Comment (cburnell) [reply ARSIS-Telemetry-GroundControl/telemetry-api/app/database.py:13]: Yup probably cruft from a mod without me actually checking this again.

File: ARSIS-Telemetry-GroundControl/telemetry-api/app/database.py [hunk 2]
```diff
@@ -1,14 +1,19 @@
+import logging
 import time
+from functools import wraps
+
 import psycopg2
+from psycopg2 import errors
```
[the remark this answers is not in the document]
Comment (cburnell) [reply ARSIS-Telemetry-GroundControl/telemetry-api/app/database.py:6]: they arent missed this with my Isort

File: ARSIS-Unity/Assets/ARSIS/Networking/Telemetry/TelemetryClient.cs [hunk 3]
```diff
@@ -5,52 +5,145 @@
...
 
-    // Update is called once per frame
-    void Update()
-    {
+    void updateServerEndpointDict(){
+        string userId = "";
+        if (registeredUser != null){
+            userId = registeredUser.id.ToString();
+        }
+        Debug.Log(userId);
```
[the remark this answers is not in the document]
Comment (cburnell) [reply ARSIS-Unity/Assets/ARSIS/Networking/Telemetry/TelemetryClient.cs:58]: Yup

File: ARSIS-Unity/Assets/ARSIS/Networking/Telemetry/TelemetryClient.cs [hunk 4]
```diff
@@ -5,52 +5,145 @@
...
     }
 
-    IEnumerator StartPollingEndpoint<Event>(string endpoint, WaitForSeconds telemetryPollingDelay){
+    IEnumerator StartPollingEndpoint<Event>(Endpoint endpoint, WaitForSeconds telemetryPollingDelay){
         while(true){
-            UnityWebRequest www = UnityWebRequest.Get(endpoint);
+            yield return telemetryPollingDelay;
+            if (registeredUser == null){
+                Debug.Log(registeredUser);
+                continue;
```
[the remark this answers is not in the document]
Comment (cburnell) [reply ARSIS-Unity/Assets/ARSIS/Networking/Telemetry/TelemetryClient.cs:131]: Oh yea have a lot of logs in this that needed to go

File: test_scripts/location_update.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    parser = argparse.ArgumentParser()
+    parser.add_argument("user_id")
+    args = parser.parse_args()
+    heading = 0
+    while True:
+        heading += 1
+        heading %= 360
+        print(heading)
+        data = {"latitude": 0, "longitude": 0, "altitude": 0, "heading": heading}
+        print(data)
```
[the remark this answers is not in the document]
Comment (cburnell) [reply test_scripts/location_update.py:16]: They really should be logs instead I'll fix it

**Review by isaacdenny — APPROVED** [review]

**Pull request merged**